<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomTrip</class>
 <widget class="QMainWindow" name="CustomTrip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBoxStartCity">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
color: black;
font-size: 14px;
}</string>
    </property>
    <property name="text">
     <string>Select the starting European city</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
color: black;
font-size: 14px;
}</string>
    </property>
    <property name="text">
     <string>Select all other European cities you wish to visit</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>181</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget{
background-color: #7cc8b1;
text-align: center;
color: white;
font-size: 10px;
border: 1px solid white;
border-radius: 5px;
margin: 10px;
padding: 10px;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_travel_efficiently">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>320</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
text-align: center;
color: white;
font-size: 14px;
background-color: rgb(176, 23, 255);
border: 1px solid black;
border-radius: 15px;
}</string>
    </property>
    <property name="text">
     <string>Travel Efficiently</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>120</y>
      <width>461</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableView{
background-color: #7cc8b1;
text-align: center;
color: white;
font-size: 10px;
border: 1px solid white;
border-radius: 5px;
margin: 10px;
padding: 10px;
}</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
